Bottom: 1f10aabdf1fb229a626c0726bb852a0c04fe690f
Top:    c0261de1bd63ed51ac4386bb4cf8cba6bf59a079
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-12 14:05:23 -0500

Refresh of add-buildbot-role

---
diff --git a/roles/buildbot-master/defaults/main.yml b/roles/buildbot-master/defaults/main.yml
index 9ce67a77..c9623933 100644
--- a/roles/buildbot-master/defaults/main.yml
+++ b/roles/buildbot-master/defaults/main.yml
@@ -1,9 +1,6 @@
 ---
 # defaults file for buildbot
 
-# host containing build assets
-asset_host: 'http://{{ansible_facts.default_ipv4.address}}/'
-
 # backup host directory
 backup_hostdir: '{{ docker_backup_dir }}/buildbot'
 
@@ -81,7 +78,7 @@ image_dir: '{{ docker_projects_dir }}/buildbot-master'
 image_name: '{{ docker_registry_username }}/buildbot'
 
 # repository URI of the image source
-image_repo: https://github.com/novatechweb/docker-buildbot-master.git
+image_repo: '{{ buildbot_image_repo | default("https://github.com/novatechweb/docker-buildbot-master.git") }}'
 
 # restoration host directory
 restore_hostdir: '{{ bacula_dest }}{{ backup_hostdir }}'
diff --git a/roles/buildbot-master/tasks/main.yml b/roles/buildbot-master/tasks/main.yml
index 8a2ac0be..d7d69e36 100644
--- a/roles/buildbot-master/tasks/main.yml
+++ b/roles/buildbot-master/tasks/main.yml
@@ -36,7 +36,7 @@
 
 - name: Grab ssh public key
   command: >
-    docker exec '{{ buildbot_container_name }}' cat /home/buildbot/.ssh/id_rsa.pub
+    docker exec '{{ container_name }}' cat /home/buildbot/.ssh/id_rsa.pub
   changed_when: false
   register: output
